lib/vagrant-service-manager/command.rb

```diff
@@ -180,9 +180,9 @@ def restart_service(service)
         command = if SCCLI_SERVICES.include? service
                     # TODO : Handle the case where user wants to pass extra arguments
                     # to OpenShift service
-                    "sccli #{service}"
+                    "sccli #{service} #{operation}"
                   else
-                    "systemctl restart #{service}"
+                    "systemctl #{operation} #{service}"
```

Comment:
As per @praveenkumar comment on `enable` and `disable`, ADB mostly provided services in the box and these operations may be defined on them.